modules/wordads/wordads.php

```diff
@@ -435,11 +451,11 @@ function get_ad( $spot, $type = 'iponweb' ) {
 				$snippet = $this->get_ad_snippet( $section_id, $height, $width, $spot, self::$SOLO_UNIT_CSS );
 				if ( $this->option( 'wordads_second_belowpost', true ) ) {
 					$section_id2 = 0 === $this->params->blog_id ? WORDADS_API_TEST_ID2 : $this->params->blog_id . '4';
-					$snippet    .= $this->get_ad_snippet( $section_id2, $height, $width, $spot, 'float:left;margin-top:0px;' );
+					$snippet    .= $this->get_ad_snippet( $section_id2, $height, $width, $spot . '2', 'float:left;margin-top:0px;' );
 				}
 			} elseif ( 'inline' === $spot ) {
 				$section_id = 0 === $this->params->blog_id ? WORDADS_API_TEST_ID : $this->params->blog_id . '5';
-				$snippet    = $this->get_ad_snippet( $section_id, $height, $width, $spot, 'mrec', self::$SOLO_UNIT_CSS );
+				$snippet    = $this->get_ad_snippet( $section_id, $height, $width, $spot, self::$SOLO_UNIT_CSS );
```

Comment:
Should we leave an empty string for the ad location?

```suggestion
				$snippet    = $this->get_ad_snippet( $section_id, $height, $width, $spot, '', self::$SOLO_UNIT_CSS );
```

Reply:
Looking at `get_ad_snippet` , the parameters the function is expecting are `$section_id, $height, $width, $location = '', $css = ''`, so the `mrec` I think is a holdover and an unrelated bug that's fixed in this PR.

Reply:
@kraftbj is correct.